src/libs/actions/IOU.ts

```diff
@@ -382,12 +382,12 @@ function setMoneyRequestTag(transactionID: string, tag: string) {
 }
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
@brunovjk  Is there any lint error if we remove this comment? If no, please remove it

Reply:
Ok, nice. We are 5 more `eslint-disable ... naming-convention`. All came from `_temporaryForRefactor`.
Will test it and remove it

Reply:
Done :D